scripts/monitor_metrics.sh

```diff
@@ -215,8 +221,7 @@ monitor_license () {
     no_license=$(grep -c "Server license: none" "$tmp_info_data")
     # Update every 60 mins
     [[ ! -f "$tmp_license_data" || $(find "$tmp_license_data" -mmin +60) ]] || return
-    $p4 license -u 2>&1 > "$tmp_license_data"
-    [[ $? -ne 0 ]] && return
+    $p4 license -u  "$tmp_license_data" 2>&1 || return
```

Comment:
Needs to save output to file!

Reply:
Changed pushed to address this - looks like a redirect operator was lost.